server/src/main/java/org/elasticsearch/index/stats/IndexingPressureStats.java

```diff
@@ -41,6 +41,8 @@ public class IndexingPressureStats implements Writeable, ToXContentFragment {
      */
     private final long lowWaterMarkSplits;
     private final long highWaterMarkSplits;
+    private final long largeOpsRejections;
+    private final long totalLargeRejectedOpsBytes;
```

Comment:
We haven't tracked the number of byte rejected in the past. I'm guessing that maybe we think this could be useful for autoscaling?

Like divide bytes rejected / ops to get an idea of what we need to scale to?

Reply:
Yes, that's the idea. But it's a bit tricky because we should make that value sticky somehow.